sydent/http/servlets/infoservlet.py

```diff
@@ -18,48 +18,30 @@
 
 import logging
 import json
-import re
-import copy
-import yaml
 
-from netaddr import IPAddress
-from sydent.db.invite_tokens import JoinTokenStore
 from sydent.http.servlets import get_args, jsonwrap, send_cors
 
-
 logger = logging.getLogger(__name__)
 
 
 class InfoServlet(Resource):
     isLeaf = True
 
-    def __init__(self, syd):
+    def __init__(self, syd, info):
```

Comment:
a docstring saying what you expect `info` to be wouldn't hurt. (likewise in InternalInfoServlet)